app/Chatkit Quickstart/MessagesDataModel.swift

```diff
@@ -59,22 +80,22 @@ class MessagesDataModel {
         let items: [MessageItem]
     }
     
-    public var delegate: MessagesDataModelDelegate?
+    var delegate: MessagesDataModelDelegate?
```

Comment:
```suggestion
    weak var delegate: MessagesDataModelDelegate?
```

Reply:
Addressed here (and in `ViewModel` also).  I have to make the `protocol`s `AnyObject` also.